app/controllers/newflow/login_signup_controller.rb

```diff
@@ -271,9 +271,10 @@ def send_reset_password_email
         },
         failure: lambda {
           user = @handler_result.outputs.user
+          email = @handler_result.outputs.email
```

Comment:
optional -- this is a law of demeter violation...not a biggie (just 2 types), if it can be @handler_result.email (even as a delegate)?